test/cmdlineTests/model_checker_contracts_only_one/err

```diff
@@ -0,0 +1,25 @@
+Warning: CHC: Assertion violation happens here.
+Counterexample:
+
+x = 0
+
+Transaction trace:
+A.constructor()
```

Comment:
Note that here it says `A.constructor()` as opposed to `B.constructor()` from the other added CLI tests. This is because in this case it ignores B as the most derived and only uses A. `B.f(0)` is still called via inheritance, but A is the deployed one.